README.md

```diff
@@ -43,6 +45,13 @@ npm run format
 npm run lint
 ```
 
+To run Cypress testing:
```

Comment:
To run Cypress testing once, use `npm run test`
To open the Cypress GUI, use `npm -w testing run cy:open`, and make sure the frontend and backend are both up as well